.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
```

Comment:
Could you please add "Please fill out relevant sections below; remove those that are unused."

Reply:
Done.